### PR TITLE
Implement Design-System Typography and Color Theme

### PR DESCRIPTION
**Pull request by yungjurick**
- Add Typography to Tailwind Config
- Add Color Theme to Tailwind Config
- Add default folder structure for deck-play subtree
- Add Framer Motion Package

### CODE REVIEW

File: packages/tailwind-config/tailwind.config.js [hunk 1]
```diff
@@ -71,8 +73,113 @@ module.exports = {
...
-  plugins: [],
+  plugins: [
+    plugin(function ({ addUtilities }) {
+      const designSystemTextStyles = {
+        // Headline
+        '.headline-1': {
+          'font-weight': 700,
+          'font-size': '28px',
+          'line-height': '39.2px',
+          'letter-spacing': '-2%'
```
Comment (poiu694) [inline packages/tailwind-config/tailwind.config.js:119]: `letter-spacing`이 `%`로는 적용이 안되는 것 같아요!

image

Comment (poiu694) [reply packages/tailwind-config/tailwind.config.js:119]: https://forum.figma.com/t/letter-spacing-should-not-be-percentage-based/3062/13

> If 0.01em is 1%, than the size would be 16/100=0.16 pixels.

**Review by poiu694 — APPROVED** [review]
고생하셨습니다 :fire:

**Review by swimjiy — APPROVED** [review]
LGTM~

File: packages/tailwind-config/tailwind.config.js (hunk 1 above)
Comment (yungjurick) [reply packages/tailwind-config/tailwind.config.js]: @poiu694 px로 다 바꿨습니당~~

**Pull request merged**